tests/XeroOauth/Request/RequestHandlerTest.php

```diff
@@ -0,0 +1,150 @@
+<?php
+namespace DarrynTen\XeroOauth\Tests\XeroOauth\Request;
+
+use DarrynTen\XeroOauth\Request\RequestHandler;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+use ReflectionClass;
+
+class RequestHandlerTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * A dummy uri for testing purposes
+     */
+    const TEST_URI = '/test-uri';
+
+    /**
+     * A config for a RequestHandler
+     * @var array
+     */
+    private $config = [
+        'key' => 'testKey',
+        'endpoint' => 'http://localhost:8082'
+    ];
+
+    /**
+     * @var RequestHandler
+     */
+    private $handler;
+
+    /**
+     * Sets up the handler object for tests
+     */
+    public function setUp()
+    {
+        $this->handler = new RequestHandler($this->config);
+    }
+
+    /**
+     * Checks if current handler object is an instance of the right class
+     */
+    public function testInstanceOf()
+    {
+        $this->assertInstanceOf(RequestHandler::class, $this->handler, 'Handler must be an instance of '.RequestHandler::class);
```

Comment:
It is marked in the contributing file that we add spaces around our dot concatenations.

We usually also prefer sprintf() 🖖